### ISSUE TITLE
Add promo image for custom campaigns

### ISSUE THREAD

**Issue by madaosoul** [vcmi/vcmi#4427]
**Describe your proposal**
When you look up a custom campaign, you see a disappointing message - that there's nothing to see. So, how about displaying a custom image? For example, a promotional art? A file specified in the campaign .json.

**How it looks right now**
![image](https://github.com/user-attachments/assets/88fdc686-7b54-49b2-b057-09f71257ac47)

**How it might look**
![image](https://github.com/user-attachments/assets/29fd0338-0364-46d0-8a28-1a40253caf60)

**Comment by madaosoul** [comment]
Oh yeah, "scenario name" should be changed to "campaign name" and "map preview" should probably be removed.

**Comment by GeorgeK1ng** [comment]
Maybe better to use something like Chronicles mapping for campaign selection screen?

<img width="1121" height="724" alt="Image" src="https://github.com/user-attachments/assets/11fbd0c4-bf22-4a13-8af8-3837a3575ba2" />

**Comment by Laserlicht** [comment]
Yes, you can use a campaignset with one campaign if you want. And use background to promote with banner.
